go/kbnm/handler.go

```diff
@@ -24,8 +25,26 @@ func handleChat(req *Request) error {
 	}
 
 	// FIXME: Get the absolute path without a filled PATH var somehow?
-	cmd := exec.Command("/usr/local/bin/keybase", "chat", "send", "--private", req.To, req.Body)
+	cmd := exec.Command("/usr/local/bin/keybase", "chat", "send", "--private", req.To)
 
-	// TODO: Check/convert status code more precisely?
-	return cmd.Run()
+	// Write message body over STDIN to avoid running up against bugs with
+	// super long messages.
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		return err
+	}
+	defer stdin.Close()
+
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+
+	if _, err = io.WriteString(stdin, req.Body); err != nil {
+		return err
+	}
+	stdin.Close()
+
+	// TODO: Check/convert status code more precisely? Maybe return stdout as
+	// part of the error if there is one?
+	return cmd.Wait()
```

Comment:
IMO checking WriteString and Close does not buy us anything, since Wait is still going to break. Pushed the version that I prefer; I think this is more equivalent to the goroutine version but without the goroutine, too.